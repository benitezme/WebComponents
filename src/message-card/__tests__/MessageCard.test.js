import React from 'react'
import renderer from 'react-test-renderer'

import { shallow } from '../../enzyme'

import MessageCard from '../index'

describe('MessageCard tests', () => {
  it('renders matching snapshot', () => {
    const tree = renderer
      .create(<MessageCard message='Renders correctly' />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders message', () => {
    const message = 'This is a test message'
    const wrapper = shallow(<MessageCard message={message} />)

    // Expect the wrapper object to be defined
    expect(wrapper.find('MessageCard')).toBeDefined()
    expect(wrapper.dive().dive().dive().find('h1'))
  })
})
