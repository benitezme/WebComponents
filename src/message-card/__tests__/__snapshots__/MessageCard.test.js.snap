// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageCard tests renders matching snapshot 1`] = `
<div
  className="MuiPaper-root-5 MuiPaper-elevation1-8 MuiPaper-rounded-6 MuiCard-root-4 MessageCard-card-1"
>
  <div
    className="MessageCard-cardDetails-2"
  >
    <div
      className="MuiCardContent-root-32 MessageCard-cardContent-3"
    >
      <h1
        className="MuiTypography-root-33 MuiTypography-headline-38 MuiTypography-gutterBottom-51 MuiTypography-alignCenter-47"
      >
        Renders correctly
      </h1>
    </div>
  </div>
</div>
`;
